models/setting.js

```diff
@@ -12,6 +12,14 @@ const SettingSchema = new Schema({
 });
 
 /**
+ * this is run once when the app starts to ensure settings are populated
+ * @return {Promise} null initialize the global settings object
+ */
+SettingSchema.statics.init = function (defaults) {
+  return this.update({id: '1'}, {$setOnInsert: defaults}, {upsert: true});
```

Comment:
We should only initialize settings when the settings object is not already initialized.

Running this every time the server starts will overwrite already existing settings with defaults. 

We need to check to see if settings are already initialized, and if they are, do not overwrite with defaults.